spec/features/collection_drafts/forms/radio_button_selectors_spec.rb

```diff
@@ -141,33 +93,38 @@
 
     context 'when selecting local coordinate system' do
       before do
-        choose 'Local'
+        choose 'Local Coordinate System'
       end
 
       it 'displays the local coordinate system fields' do
         expect(page).to have_css('.local-coordinate-system-fields')
       end
 
       it 'does not display the other form fields' do
-        expect(page).to have_no_css('.geographic-coordinate-system-fields')
+        expect(page).to have_no_css('.horizontal-data-resolution-fields')
       end
     end
 
     context 'when switching between options after filling in form data' do
       before do
-        choose 'Geographic'
-        select 'Meters', from: 'Geographic Coordinate Units'
-        fill_in 'Latitude Resolution', with: '42.0'
-        fill_in 'Longitude Resolution', with: '43.0'
+        choose 'Horizontal Data Resolution'
+        check 'Gridded Resolutions'
+        select 'Meters', from: 'Unit'
+        fill_in 'X Dimension', with: '42.0'
+        fill_in 'Y Dimension', with: '43.0'
 
-        choose 'Local'
-        choose 'Geographic'
+        choose 'Local Coordinate System'
+        choose 'Horizontal Data Resolution'
       end
 
       it 'clears the form data' do
-        expect(page).to have_field('Geographic Coordinate Units', with: '')
-        expect(page).to have_field('Latitude Resolution', with: '')
-        expect(page).to have_field('Longitude Resolution', with: '')
+        expect(page).to have_no_checked_field('Gridded Resolutions')
+
+        check 'Gridded Resolutions'
+
+        expect(page).to have_field('Unit', with: '')
+        expect(page).to have_field('X Dimension', with: '')
+        expect(page).to have_field('Y Dimension', with: '')
```

Comment:
Do we want a test for the data clearing with the checkboxes being toggled on and off?

Reply:
yes, good catch

Reply:
added it in the test for checkboxes